packages/dd-trace/src/dogstatsd.js

```diff
@@ -104,3 +128,23 @@ class Client {
 }
 
 module.exports = Client
+
+function requestQueue (queue, options, callback) {
```

Comment:
Let's keep the export as the last line so that it's easy to find.

Reply:
Also, why is this function needed?

Reply:
If we have multiple queued payloads, we'll send them all sequentially, like we do with UDP. This is to make that easy for HTTP.

Reply:
Still not sure I understand, why is it not enough to simply send the request? The queueing happens externally.